### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,16 +18,22 @@ The following instruction assume your are using Python 2 and an Ubuntu installat
 
 ## Dependencies
 
-Pymavlink requires two dependencies :
+Pymavlink requires 3 dependencies :
```
Comment (peterbarker) [inline README.md]: You only list two... and on Linux python-dev (e.g.) is also a dependency due to the mavnative stuff.

File: README.md [hunk 2]
```diff
@@ -18,16 +18,22 @@ The following instruction assume your are using Python 2 and an Ubuntu installat
...
 
-    - future : for python 2 and python 3 interoperability
-    - lxml : for checking and parsing xml file
+    - future : for python 2 and python 3 interoperability (http://python-future.org/)
+    - lxml : for checking and parsing xml file (http://lxml.de/installation.html)
 
 ### On linux
 
-Using pip you can install the required dependencies
+Lxml require some dependencies too
```
Comment (peterbarker) [inline README.md]: capitalisation, and should be "requires", not "require".

Probably better to be "lxml has some additional dependencies"

File: README.md [hunk 3]
```diff
@@ -18,16 +18,22 @@ The following instruction assume your are using Python 2 and an Ubuntu installat
...
 
 ```bash
+sudo apt-get install libxml2-dev libxslt-dev python-dev
```
Comment (peterbarker) [inline README.md]: Should mention this is for debian-derived systems and the user should adjust for their particular package manager.

**Comment by khancyr** [comment]
Rebased and correct. 
Should solve a part of https://github.com/ArduPilot/pymavlink/issues/41

**Comment by peterbarker** [comment]
Merged, thanks

**Pull request merged**